src/pages/Tracking/Tracking.jsx

```diff
@@ -106,6 +113,37 @@ class Tracking extends Component {
 		}
 	};
 
+	updateState = (index, value, type) => {
```

Comment:
I like that using `index` everywhere is easier for looking up the deer in the array. I wonder though if there's any chance it could backfire if a new deer is pushed to the front of the list maybe? Do you see anything like that happening?